docs/proposal/203-duffle-build.md

```diff
@@ -1,5 +1,5 @@
 # Duffle Build
 
-This document describes how `duffle build` works, and how it uses the duffle build file: `duffle.json, duffle.yaml, duffle.toml`.
+This document describes how `duffle build` works, and how it uses the duffle build configuration file: `duffle.json, duffle.yaml, duffle.toml`.
 
-`duffle build` take a path to a directory that contains a duffle build file (`duffle.json, duffle.toml, or duffle.yaml`) to build a Cloud Native Application Bundle (CNAB). In the process, it also builds all of the invocation images specified in the duffle build file.
+`duffle build` take a path to a directory that contains a duffle build configuration file (`duffle.json, duffle.toml, or duffle.yaml`) to build a Cloud Native Application Bundle (CNAB). In the process, it also builds all of the invocation images specified in the duffle build configuration file.
```

Comment:
@benpatt Unrelated to this PR, but I wonder whether `duffle.toml` or `duffle.yaml` work? It would be worth investigating and raising a bug if necessary.

Reply:
Addressed in this pull request: https://github.com/deislabs/duffle/pull/803